app/main.py

```diff
@@ -1,6 +1,20 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+FORMAT = "json"
+BASE_URL = "http://api.weatherapi.com/v1"
+CITY_NAME = "Paris"
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    url = f"{BASE_URL}/current.{FORMAT}?key={API_KEY}&q={CITY_NAME}"
+    response = requests.get(url)
+    print(response.status_code, response.json())
```

Comment:
Printing `response.json()` without checking if the request was successful can lead to errors if the response is not JSON (e.g., in case of an error response). Consider checking `response.ok` or `response.status_code` before attempting to parse the JSON.